index.go

```diff
@@ -295,39 +300,42 @@ func main() {
 		r.Get("/search/geo/:latitude/:longitude", dealers_ctlr.SearchLocationsByLatLng)
 	})
 
+	//Creating, updating, and deleting FAQs are done in GoAdmin directly
```

Comment:
If that's the case, these routes seem like they may be good candidates for deprecating. I've been creating tickets similar to API-105 for endpoints to keep track of routes that might not be used anymore. Sometime down the road when we get enough enpoints (and free time ;) ) we were going to evaluate further and see if we can actually deprecate/remove them